core/kits/documents/kit.php

```diff
@@ -92,4 +94,29 @@ protected function get_post_statuses() {
 			'publish' => __( 'Published', 'elementor' ),
 		];
 	}
+
+	public function add_repeater_row( $control_id, $item ) {
+		$meta_key = PageManager::META_KEY;
+		$document_settings = $this->get_meta( $meta_key );
+
+		if ( ! $document_settings ) {
+			$document_settings = [];
+		}
+
+		if ( ! isset( $document_settings[ $control_id ] ) ) {
+			$document_settings[ $control_id ] = [];
+		}
+
```

Comment:
You can skip the first check. The second one will do them both

Reply:
https://github.com/elementor/elementor/pull/11622/commits/5a93721d550eebb83830727a4a291009e80f83b4 fails.

Reply:
What kind of failure?

Reply:
`array_merge(): Argument #1 is not an array`
https://github.com/elementor/elementor/runs/780041780

Reply:
This fail is not related to this comment

Reply:
> This fail is not related to this comment

i just tried right now, to remove those lines
```
		if ( ! $document_settings ) {
			$document_settings = [];
		}
```
and tests fails.